app/main/forms/frameworks.py

```diff
@@ -30,21 +29,21 @@ class ReuseDeclarationForm(FlaskForm):
 
 
 class OneServiceLimitCopyServiceForm(FlaskForm):
-
-    copy_service = DMBooleanField(
-        "Do you want to reuse your previous {lot_name} service?",
-        false_values=("False", "false", ""),
-        widget=DMSelectionButtonBase(type="boolean"),
+    copy_service = RadioField(
+        id="input-copy_service-1",  # TODO: change to input-copy_service when on govuk-frontend~3
+        choices=[("yes", "Yes"), ("no", "No")],
+        validators=[InputRequired("Select yes if you want to reuse your previous service")],
```

Comment:
Odd that it wasn't compulsory to select a response on a radio. Definitely better this way.